apps/webstore/src/app/pages/home/home.component.html

```diff
@@ -32,18 +33,14 @@ <h2 i18n="@@reviews">Reviews</h2>
             />
             <span class="trustpilot-rating-number">4.5 <span class="trustpilot-rating-text">out of 5</span> </span>
         </div>
-        <div
-            class="trustpilot-widget"
-            data-locale="en-US"
-            data-template-id="5419b6a8b0d04a076446a9ad"
-            data-businessunit-id="6278fc669c36a16dd3a56788"
-            data-style-height="24px"
-            data-style-width="100%"
-            data-theme="light"
-            data-min-review-count="10"
-        >
-            <a href="https://www.trustpilot.com/review/treecreate.dk" target="_blank" rel="noopener">Trustpilot</a>
-        </div>
+        <img
```

Comment:
This isn't an actual link so the user can't click on it and see the trustpilot page. You need to wrap it in a <a> element (I think)